Committees/Chapter Committees/CFEC/CFEC.adoc

```diff
@@ -0,0 +1,37 @@
+= Chapter Firearms Education Committee (CFEC)
+// Title of committee goes here
+:toc:
+
+== Composition
+// Use this space to describe who the committee is composed of
+The CFEC is composed of chapter members who meet the requrements listed below in the requirements section, and have applied for the committee. The committee is lead by the <<CFED.adoc#,Chapter Firearms Education Director (CFED)>>
+
+== Requirements
+// This section should describe the requirements that need to be met to join  this committee. For national or regional committees, include this section:
+// Joining this committee requires you fulfill the requirements listed here (link to relevant chapter committee director)
+To join the CFEC, you must complete the following PLTCOM courses:
+
+* PLTCOM Intro to Scientific Socialism
```

Comment:
We should standardize the course titles. Intro or Introduction?

Reply:
Also, we probably don't need to double up on PLTCOM here

Reply:
Actually, I'd like course _numbers_ , but I think that's @Joedirte1917's department.